install/terraform/modules/gke/cluster.tf

```diff
@@ -23,24 +23,31 @@ data "google_client_config" "default" {}
 # Run `terraform taint null_resource.test-setting-variables` before second execution
 resource "null_resource" "test-setting-variables" {
   provisioner "local-exec" {
-    command = "${format("echo Current variables set as following - name: %s, project: %s, machineType: %s, initialNodeCount: %s, zone: %s",
+    command = <<EOT
+      ${format("echo Current variables set as following - name: %s, project: %s, machineType: %s, initialNodeCount: %s, network: %s, zone: %s",
       var.cluster["name"], var.cluster["project"],
-      var.cluster["machineType"], var.cluster["initialNodeCount"],
-    var.cluster["zone"])"
+      var.cluster["machineType"], var.cluster["initialNodeCount"], var.cluster["network"],
+      var.cluster["zone"])}
+    EOT
   }
 }
 resource "google_container_cluster" "primary" {
   name     = var.cluster["name"]
   location = var.cluster["zone"]
   project  = var.cluster["project"]
-  provider = google-beta
+  network  = var.cluster["network"]
+  provider = google
```

Comment:
There's no need to specify this. The provider is effectively determined by string parsing the resource name so this resource will default to `google` if not explicitly overridden. 

Also FWIW we don't consider the beta provider as an unstable provider. Instead we just provide it as a way for people to call the beta GCP APIs if they are interested in using features/fields that are only available in the `beta version of the product. There are a few more [details about the provider versions here](https://www.terraform.io/docs/providers/google/guides/provider_versions.html). That being said if you aren't using beta features then there is no need to use the beta provider.

Reply:
Thanks for the cleanup and explanation :+1: removing the reference.

My concern with the google beta api surface isn't stable, was that if we happen to inadvertently use a beta feature. And since we aren't, there doesn't seem to be a good reason to take the risk.